server/src/internalClusterTest/java/org/opensearch/discovery/ClusterDisruptionIT.java

```diff
@@ -386,7 +386,7 @@ public void onFailure(Exception e) {
         );
 
         if (isolatedNode.equals(nonClusterManagerNode)) {
-            assertNoMaster(nonClusterManagerNode);
+            assertNoClusterManager(nonClusterManagerNode);
         } else {
```

Comment:
Just putting it here the variables remaining in this file: `getMasterName`, `startMasterOnlyNodes`. Feel free to link if you any PR already.

Reply:
Thanks for your review! 👍
The [getMasterName()](https://github.com/opensearch-project/OpenSearch/blob/2.0.0-rc1/test/framework/src/main/java/org/opensearch/test/InternalTestCluster.java#L1948) and [startMasterOnlyNodes()](https://github.com/opensearch-project/OpenSearch/blob/2.0.0-rc1/test/framework/src/main/java/org/opensearch/test/InternalTestCluster.java#L2149) are both public method, which are published to [maven](https://mvnrepository.com/artifact/org.opensearch.test/framework), I will rename them in the next phase (issue https://github.com/opensearch-project/OpenSearch/issues/1684).